### PR TITLE
WorldMapPlugin: Add tooltip to minigame icons

### PR DESCRIPTION
**Pull request by MagicfTail**
Add tooltip showing the name of the minigame on the worldmap

### CODE REVIEW

**Comment by MESLewis** [comment]
The title / commit message should probably say "minigame" instead of "minimap"

**Comment by MagicfTail** [comment]
Rebased

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/MinigameLocation.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+	private final String tooltip;
+	private final WorldPoint location;
+	private final MinigamePoint minigamePoint;
+
+	MinigameLocation(String tooltip, WorldPoint location)
+	{
+		this.tooltip = tooltip;
+		this.location = location;
+		this.minigamePoint = new MinigamePoint(tooltip, location);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/MinigameLocation.java]: Just create the MinigamePoint from MinigameLocation no need to store it inside the location

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java [hunk 2]
```diff
@@ -172,6 +185,12 @@ protected void startUp() throws Exception
 				.map(value -> new AgilityShortcutPoint(value, BLANK_ICON))
 				.forEach(worldMapPointManager::add);
 		}
+		if (config.minigameTooltip())
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java]: Empty line above

**Comment by MagicfTail** [comment]
Like so?

**Pull request merged**